src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -51,7 +51,7 @@ namespace System.Management.Automation
     ///     - There is no ITypeDescriptorContext.
```

Comment:
@iSazonov, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Language Primitive Tests.Test convertion with .Net Core intrinsic type convertor
```powershell
Expected strings to be the same, but they were different.
Expected length: 20
Actual length:   21
Strings differ at index 20.
Expected: 'http://test.site.com'
But was:  'http://test.site.com/'
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Api\LanguagePrimitive.Tests.ps1: line 44
44:         $result.AbsoluteUri | Should -Be 'http://test.site.com'
```
Remoting loopback tests.Can connect to all disconnected sessions by name
```powershell
Cannot bind parameter 'ResourceURI'. Cannot convert value "Shell" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 249
249:     Connect-PSSession @parameters

```
New-ModuleManifest tests.Relative URIs are not allowed
```powershell
Cannot convert value "../foo" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Module\NewModuleManifest.Tests.ps1: line 115
115:         $testUri = [Uri]"../foo"

```
WSMan Config Provider.New-Item and Remove-Item tests.New-Item and Remove-Item for a plugin using parameters
```powershell
Cannot bind parameter 'Resource'. Cannot convert value "foo" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 338
338:                     -SDKVersion 2 -Resource foo -Capability shell -XMLRenderingType text -Path WSMan:\localhost\plugin `

```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Configuration.ConfigurationManager.dll; SideIndicator==>}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```